ui/apps/platform/src/services/ComplianceResultsService.ts

```diff
@@ -85,11 +87,24 @@ export function getComplianceProfilesStats(): Promise<ListComplianceProfileScanS
  * Fetches the profile check results.
  */
 export function getComplianceProfileResults(
-    profileName: string
+    profileName: string,
+    page: number,
+    pageSize: number
 ): Promise<ListComplianceProfileResults> {
+    let offset: number | undefined;
+    if (typeof page === 'number' && typeof pageSize === 'number') {
+        offset = page > 0 ? page * pageSize : 0;
+    }
+    const queryParameters = {
+        query: {
+            pagination: { offset, limit: pageSize },
```

Comment:
What happens with the API when offset is actually `undefined` and/or `pageSize` has an invalid value?

Reply:
I feel like we've done a good enough job of typing these all out to where I'm not concerned

Reply:
I see the logic was updated. You can disregard this then.